api/envoy/extensions/compression/zstd/compressor/v3/zstd.proto

```diff
@@ -34,25 +34,29 @@ message Zstd {
     BTULTRA2 = 9;
   }
 
-  // Set compression parameters according to pre-defined cLevel table.
+  // Set compression parameters according to pre-defined compression level table.
   // Note that exact compression parameters are dynamically determined,
-  // depending on both compression level and srcSize (when known).
+  // depending on both compression level and source content size (when known).
   // Value 0 means default, and default level is 3.
   // Setting a level does not automatically set all other compression parameters
   // to default. Setting this will however eventually dynamically impact the compression
   // parameters which have not been manually set. The manually set
   // ones will 'stick'.
   google.protobuf.UInt32Value compression_level = 1;
 
-  // A 32-bits checksum of content is written at end of frame (default:0)
+  // A 32-bits checksum of content is written at end of frame, If not set, defaults to false.
   bool enable_checksum = 2;
 
   // The higher the value of selected strategy, the more complex it is,
   // resulting in stronger and slower compression.
   // Special: value 0 means "use default strategy".
   Strategy strategy = 3 [(validate.rules).enum = {defined_only: true}];
 
-  // Dictionary for compression.
+  // A dictionary for compression. zstd offers dictionary compression, which greatly improves
+  // efficiency on small files and messages. Each dictionary will be generated with a dictionary ID
+  // that can be used to search the same dictionary during decompression.
+  // Please refer to ztsd manual to train a specific dictionary for compression:
+  // https://github.com/facebook/zstd/blob/dev/programs/zstd.1.md#dictionary-builder
```

Comment:
Please make this into a proper link, same below.

Reply:
Hi, @mattklein123, I think I didn't get the proper link's mean. Do you mean the we should use a html website manual 
 like below:
```
https://www.freebsd.org/cgi/man.cgi?query=zstd&sektion=1&format=html#end(It is from third part page, and can't link to the correct position)
https://www.manpagez.com/man/1/unzstd/zstd-1.3.4.php(Same with upper)
```
Or how a bout this?
```
https://github.com/facebook/zstd/blob/dev/doc/zstd_compression_format.md#dictionary-format(Introduce the dictionary format, relate to how to train a dictionary)
```
Could you leave more detail here? Thanks!

Reply:
Sorry what I mean is make these a sphinx link so they show up in the docs as links:

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

We generate docs from these files.

/wait

Reply:
Got it and thanks! Done.